### ISSUE TITLE
Incorrect WebDriver downloaded for Microsoft Edge

### ISSUE THREAD

**Issue by peterjohngriffin** [bonigarcia/webdrivermanager#347]
**Description of the problem**: 
This issue is almost exactly the same as issue #339, which was correctly closed, having been fixed as part of WebDriverManager v3.5.0.

I require Microsoft Edge WebDriver Version 6.17134, but Version 4.15063 is downloaded from here: https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

I would guess that this error, like issue #339, is caused by a change in the layout of Microsoft's website.

**WebDriverManager version**: 
LATEST, therefore v3.6.0 currently

**Operating System**: 
Windows 10 64 bit

**Microsoft Edge version**:
Microsoft Edge 42.17134.1.0
Microsoft EdgeHTML 17.17134

**Comment by bonigarcia** [comment]
This is related to #338, recently re-opened. I already commit a patch solving this issue. It will be available for the next version of WebDriverManager.

**Comment by bonigarcia** [comment]
This should be fixed in version 3.6.1, recently released.

**Comment by acenturion** [comment]
Hi!,  
i changed the version to 3.6.1 and i have the same issue, i need to do something else?

**Comment by bonigarcia** [comment]
@acenturion Which version of Edge are you trying to drive with Selenium?

Also, I would need to see the WebDriverManager logging to trace the problem. For example:

```
2019-06-13 09:54:35 [main] DEBUG i.g.bonigarcia.wdm.WebDriverManager.getBrowserVersion(216) -- Edge Dev (based on Chromium) version 76 found
2019-06-13 09:54:35 [main] DEBUG io.github.bonigarcia.wdm.Preferences.checkValidity(93) -- Removing preference edge76=76.0.168.0 (expired on 2019-06-05 18:09:27)
2019-06-13 09:54:35 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.detectDriverVersionFromBrowser(575) -- Using msedgedriver 76.0.168.0 (since Microsoft Edge 76 is installed in your machine)
2019-06-13 09:54:35 [main] DEBUG io.github.bonigarcia.wdm.Preferences.putValueInPreferencesIfEmpty(66) -- Storing preference edge76=76.0.168.0 (valid until 2019-06-14 09:54:35)
2019-06-13 09:54:35 [main] DEBUG i.g.bonigarcia.wdm.WebDriverManager.manage(530) -- Driver msedgedriver 76.0.168.0 found in cache
2019-06-13 09:54:35 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.exportDriver(1050) -- Exporting webdriver.edge.driver as C:\Users\boni\.m2\repository\webdriver\edgedriver\win64\x64\76.0.168.0\msedgedriver.exe
```

**Comment by acenturion** [comment]
> 
> 
> @acenturion Which version of Edge are you trying to drive with Selenium?
> 
> Also, I would need to see the WebDriverManager logging to trace the problem. For example:
> 
> ```
> 2019-06-13 09:54:35 [main] DEBUG i.g.bonigarcia.wdm.WebDriverManager.getBrowserVersion(216) -- Edge Dev (based on Chromium) version 76 found
> 2019-06-13 09:54:35 [main] DEBUG io.github.bonigarcia.wdm.Preferences.checkValidity(93) -- Removing preference edge76=76.0.168.0 (expired on 2019-06-05 18:09:27)
> 2019-06-13 09:54:35 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.detectDriverVersionFromBrowser(575) -- Using msedgedriver 76.0.168.0 (since Microsoft Edge 76 is installed in your machine)
> 2019-06-13 09:54:35 [main] DEBUG io.github.bonigarcia.wdm.Preferences.putValueInPreferencesIfEmpty(66) -- Storing preference edge76=76.0.168.0 (valid until 2019-06-14 09:54:35)
> 2019-06-13 09:54:35 [main] DEBUG i.g.bonigarcia.wdm.WebDriverManager.manage(530) -- Driver msedgedriver 76.0.168.0 found in cache
> 2019-06-13 09:54:35 [main] INFO  i.g.bonigarcia.wdm.WebDriverManager.exportDriver(1050) -- Exporting webdriver.edge.driver as C:\Users\boni\.m2\repository\webdriver\edgedriver\win64\x64\76.0.168.0\msedgedriver.exe
> ```

I fixed the problem, was the driver downloaded in 3.6.1 is not compatible with my local edge.

i went to .m2/repository/webdriver/MicrosoftWebDriver/6.17134 
- delet the current driver
- put the new driver in folder 

i have this stacktrace: 

`Timed out waiting for driver server to start.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'VAARINPDWKPM7', ip: '10.10.6.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_201'
Driver info: driver.version: Driver
org.openqa.selenium.WebDriverException: Timed out waiting for driver server to start.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'VAARINPDWKPM7', ip: '10.10.6.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_201'
Driver info: driver.version: Driver
	at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:202)
	at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:188)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:96)
	at Helpers.Driver.createDriverNew(Driver.java:45)`

**Comment by peterjohngriffin** [comment]
This appears to be an issue again in the latest version of WebDriverManager.

**Comment by bonigarcia** [comment]
Solved in release 3.6.2, just released.
